idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt

```diff
@@ -163,7 +164,10 @@ object KDocRenderer {
         // Avoid wrapping the entire converted contents in a <p> tag if it's just a single paragraph
         val maybeSingleParagraph = markdownNode.children.singleOrNull { it.type != MarkdownTokenTypes.EOL }
         return if (maybeSingleParagraph != null && !allowSingleParagraph) {
-            maybeSingleParagraph.children.joinToString("") { it.toHtml() }
+            val lineSeparator = LineSeparator.getSystemLineSeparator().separatorString
```

Comment:
Source code in IntelliJ always uses `\n` as a line separator. No need to access the system line separator here.